gooddata-java/src/main/java/com/gooddata/sdk/service/project/ProjectService.java

```diff
@@ -124,6 +124,14 @@ public ProjectService(final RestTemplate restTemplate, final AccountService acco
         return new PageBrowser<>(startPage, page -> listProjects(getProjectsUri(userId, page)));
     }
 
+    public PageBrowser<Project> listProjects(final Account account, final PageRequest startPage) {
```

Comment:
@liry please check here

Reply:
I would add one more "convenience" method without `startPage` argument the same way we have it for current user (without account arg).

Reply:
And please add javadoc.